packages/env/lib/env.js

```diff
@@ -223,19 +242,40 @@ module.exports = {
  * Initializes the local environment so that Docker commands can be run. Reads
  * ./.wp-env.json, creates ~/.wp-env, and creates ~/.wp-env/docker-compose.yml.
  *
+ * @param {Object}  options
+ * @param {Object}  options.spinner A CLI spinner which indicates progress.
+ * @param {boolean} options.debug   True if debug mode is enabled.
+ *
  * @return {Config} The-env config object.
  */
-async function initConfig() {
+async function initConfig( { spinner, debug } ) {
 	const configPath = path.resolve( '.wp-env.json' );
 	const config = await readConfig( configPath );
+	config.debug = debug;
```

Comment:
Why add this?

Reply:
The config is already passed everywhere, so it saves us from adding new parameters to every function.

Reply:
Fair enough. It's maybe worth adding `config` to the JSDoc type definition for `Config`.

Reply:
Done